app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -47,7 +48,64 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+              rows={4}
+            />
+            <Select
+              isClearable
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Assigned Contacts"
+              name="ValueList"
```

Comment:
Find a better name appropriate to the business logic. This is a technical term.

Reply:
Done